addons/stretcher/stringtable.xml

```diff
@@ -106,6 +107,7 @@
             <Dutch>Trek brancard terug</Dutch>
             <Spanish>Recoger Camilla</Spanish>
             <Japanese>担架を収容する</Japanese>
+            <German>Trage aufheben</German>
```

Comment:
I assume this is about the winching feature of the helistretcher, which would mean "Trage einholen" could work better